composer.json

```diff
@@ -39,7 +39,8 @@
 		}
 	],
 	"require": {
-		"php": ">=5.3.3",
+		"php": ">=5.4",
+		"phpbb/phpbb": ">=3.2.0@dev",
```

Comment:
We'll continue this discussion on the composer repository ticket and this can always be changed at a later date.